cfdm/data/netcdfarray.py

```diff
@@ -454,6 +472,41 @@ def get_mask(self):
         """
         return self._get_component("mask")
 
+    def get_missing_values(self):
+        """The missing value indicators from the netCDF variable.
+
+        .. versionadded:: (cfdm) 1.10.0.3
+
+        :Returns:
+
+            `dict` or `None`
+                The missing value indicators from the netCDF variable,
+                keyed by their netCDF attribute names. An empty
+                dictionary signifies that no missing values are given
+                in the file. `None` signifies that the missing values
+                have not been set.
+
+        **Examples**
+
+        >>> a.get_missing_values()
+        None
+
+        >>> a.get_missing_values()
+        {}
+
+        >>> a.get_missing_values()
+        {'missing_value': 1e20, 'valid_range': (-10, 20)}
+
+        >>> a.get_missing_values()
+        {'valid_min': -999}
```

Comment:
A bit pedantic, but if we're not indicating any change in `a` it might avoid any potential confusion if we use different variable names for each example, e.g:

```suggestion
        >>> a.get_missing_values()
        None

        >>> b.get_missing_values()
        {}

        >>> c.get_missing_values()
        {'missing_value': 1e20, 'valid_range': (-10, 20)}

        >>> d.get_missing_values()
        {'valid_min': -999}
```